client/my-sites/sidebar-unified/use-site-menu-items.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestAdminMenu } from '../../state/admin-menu/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getAdminMenu } from 'state/admin-menu/selectors';
+
+function useSiteMenuItems() {
```

Comment:
how would you feel changing it to
`const useSiteMenuItems = () => {`
in order to be more in par with other implementations?

Reply:
Done.